index.js

```diff
@@ -122,21 +138,21 @@ module.exports = {
     strict: 0,
     'object-curly-spacing': 0,
     'operator-linebreak': [1, 'after'],
-    quotes: [0, 'single', 'avoid-escape'],
+    quotes: [1, 'single', 'avoid-escape'],
```

Comment:
moving this to a warn because `eslint --fix` will not fix this properly if we're ignoring it. in particular, this allows eslint to auto-fix
```
const someObj = {
  someKey: someValue,
  'someSingleQuotedKey': someValue,
};
```
to
```
const someObject = {
  'someKey': someValue,
  'someSingleQuotedValue': someValue,
}
```
(without this, it will make it double-quoted via `quote-props` and won't fix it via the `quotes` rule)